pkg/coscheduling/coscheduling.go

```diff
@@ -215,3 +244,18 @@ func (cs *Coscheduling) rejectPod(uid types.UID) {
 	}
 	waitingPod.Reject(Name)
 }
+
+func (cs *Coscheduling) getStateKey(pod *v1.Pod) framework.StateKey {
+	return framework.StateKey(fmt.Sprintf("%v-%v", cs.Name(), core.GetNamespacedName(pod)))
```

Comment:
CycleState is per pod, so you don't need to use Pod's name as the key. Usually we just use "PreFilter" + plugin name as the key.

(Ref: VolumeBinding's example: https://github.com/kubernetes/kubernetes/blob/42c00bc523c54ea72177af33a806e611850e44c9/pkg/scheduler/framework/plugins/volumebinding/volume_binding.go#L42)